Provider, Publisher, Invoice Issuer, Datagenerator Consistency Refactor (#834)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/serviceprovider.md

# Service Provider

A Service Provider is the entity that makes the [*resources*](#glossary:resource) or [*services*](#glossary:service) available for purchase or consumption.  It is commonly used for cost analysis and reporting scenarios.

The Service Provider column adheres to the following requirements:

* The Service Provider column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset).
* This column MUST be of type String and MUST NOT contain null values.

> See [Appendix: Origination of cost data](#originationofcostdata) section for examples of Service Provider values that can be used for various purchasing scenarios.

Reviewer: - **Editorial:**  The reference to the appendix (both the title and the URL) needs to be updated
- **Note:** The examples are no longer limited to purchasing scenarios?  

```suggestion
See [Appendix: Entity Identification Examples](#entityidentification) section for examples of Service Provider values for various use case scenarios.
```
